docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Federation in the Apollo Platform
+description: Use all of the features of the Apollo Platform with a federated GraphQL architecture
+---
+[//]: # (Description: Introductory content, discussing the overall model of the federation platform, why you would want to register partial schemas, and framing the concept of thinking about running federation as a coordination and collaboration problem to motivate the tools we've built)
+[//]: # (Assignee: Chang)
+[//]: # (Reviewer: Caydie, James)
+
+If you're running your GraphQL infrastructure using GraphQL services that comply with the [federation spec](https://www.apollographql.com/docs/apollo-server/federation/federation-spec/), there are a variety of things you'll want to consider when using the Apollo GraphQL platform. The key difference between running GraphQL with a monolithic server instead of running multiple services orchestrated by a [gateway](https://www.apollographql.com/docs/apollo-server/federation/implementing/#running-a-gateway) is that your GraphQL schema is dynamically defined by [partial schemas](<LINK_TO_GLOSSARY>), which compose to form an overall schema for your graph. Since the GraphQL schema is the central hub of the data in your graph, it's important to **track each implementing service's partial schemas** in order to ensure that they are able to compose into a valid schema, and also to [validate](<LINK_TO_SCHEMA_VALIDATION>) that the changes any service makes do not affect the [operations made by clients](LINK_TO_METRICS).
+
+... More overview stuff
+
+## Inspecting your Graph
+> Using the `apollo service:list` command or the UI to understand your graph
+
+To understand the implementing services that make up your graph, you can use both the Apollo command line interface as well as the Apollo Platform.
+
+Use `apollo service:list` in the command line to see a snapshot of the services that make up your graph, including their endpoints and when they were last updated.
+
+Here's what running `apollo service:list` will look like:
+
+```
+~$ apollo service:list
+  ✔ Loading Apollo Project
+  ✔ Fetching list of services for graph service-list-federation-demo
+
+name       URL                            last updated
+─────────  ─────────────────────────────  ────────────────────────
```

Comment:
the labels above are still in a [PR](https://github.com/apollographql/apollo-tooling/pull/1377/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e) but I realized they are necessary when writing this documentation: